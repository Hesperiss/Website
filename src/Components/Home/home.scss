//fonts
@import url('https://fonts.googleapis.com/css?family=Montserrat|Poppins&display=swap');

//theme colors
$theme-light: #58a5f0;
$theme-medium: #0277bd;
$theme-dark: #004c8c;
$theme-gray: #242f3e;
$very-light-gray: #eeeeee;

.homePage {

  background-color: $theme-gray;
  display: flex;
  flex-grow: 1;
  flex-flow: column wrap;
  place-items: center;
  place-content: center;
  place-self: center;
  width: 100%;
  min-height: 100vh;
  height: 100%;
  .blueCurveWrapper{
    z-index: 1;
  }

  .root{
    background-color: $theme-dark;
  }
}

.homePageContents {

  z-index: 10;
  color: $theme-medium;
  align-self: center;
  place-content: center;
  place-items: center;
  box-sizing: border-box;
  display: flex;
  flex-flow: column;
  //flex-grow: 1;
  //flex-shrink: 1;
  width: 100%;
  text-align: center !important;

  h1 {
    color: white;
    font-family: Montserrat, sans-serif;
    text-transform: uppercase;
    font-weight: bold;
    font-size: 6em;
  }

  h2 {
    text-transform: none;
    margin: 20px;
    font-family: Poppins, sans-serif;
    font-weight: lighter;
    color: $very-light-gray;
    font-size: 24px;
  }

  .descBoxWrapper {

    #docImage{
      @media only screen and (min-width: 600px) {
        width: 20%;
        min-height: 300px;
      }
      width: 100%;
      min-height: 200px;
      background-size: contain;
      background-repeat:no-repeat;
      background-position: center;
      background-image: url("../../Images/doc_clipboard.png");
    }

    @media only screen and (min-width: 768px) {
      flex-flow: row wrap;
      width: 100%;
    }
    flex-flow: column;
    width: 100%;
    place-items: center;
    place-content: center;
    margin: 50px 0;
    align-self: center;
    justify-self: center;
    display: flex;

  }

  .descBox {
    @media only screen and (min-width: 768px) {
      align-self: stretch;
      flex-flow: row wrap;
      width: 20%;
      margin: 10px;
    }
    min-width: 200px;
    width: 80%;
    margin: 50px 0;
    padding: 25px;
    background-color: $theme-light;
    box-shadow: 0px 0px 12px 3px rgba(0, 0, 0,.3);
    overflow-x: auto;
    border-radius: 10px;
    color: $theme-dark;

    .rhombus, .rhombusWrapper {
      background-color: white;
    }

    &:hover {
      transition: all 0.1s ease-in;
      box-shadow: 0px 0px 12px 3px rgba(0, 0, 0,.5);
      text-decoration: none;
      p {
        color: $theme-gray;
      }
      h3 {
        color: $theme-dark;
      }
      .rhombus, .rhombusWrapper {
        background-color: $theme-medium;
      }
    }

    p {
      color: $theme-gray;
      text-align: center;
      font-size: 16px;
    }
  }
}

.landingLink{
  z-index: 10;
  place-content: center;
  place-items: center;
  display: flex;
  flex-flow: row wrap;
  font-family: Montserrat, Poppins, Arial, sans-serif;
  font-weight: bold;
  font-size: 12px;
  text-align: center;
  padding: 15px 50px;
  color: white;
  background-size: 200% 100%;
  background-image: linear-gradient(to right,  $theme-dark 50%, $theme-light 50%);
  transition: background-position 0.5s;
  border-radius: 100px;

  &:hover {
    background-position: -100% 0%;
    transition: all 0.5s ease-in;
    box-shadow: inset 0 0 0 3px white;
    .icon {
      color: $theme-gray;
    }
  }
  h3 {
    font-family: Poppins, Arial, sans-serif;
    text-decoration: none !important;
    font-style: normal;
  }
}




