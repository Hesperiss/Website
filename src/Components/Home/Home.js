import React from 'react';
import './home.scss'
import {FaCommentMedical, FaMapMarkedAlt} from "react-icons/all";
import {Helmet} from "react-helmet";
import ChatBotWidget from '../Chat/ChatBotWidget';
import {toggleWidget} from 'react-chat-widget';
import {Link} from "react-router-dom";
import {FormattedMessage} from "react-intl";


export default function HomePage() {

    const displayFeaturesDescriptionBoxes = () => {

        return (
            <div className="homePageContents">

                <div className={"textWrapper"}>
                    <h1>Kwili</h1>
                    <h2>
                        <FormattedMessage
                            id={"Home.Tagline"}
                            defaultMessage={"Un accompagnement personnalisé pour vous conseiller dans votre urgence"}
                        />
                    </h2>
                </div>

                <div className={"descBoxWrapper"}>
                    <div className="descBox" id="chat" onClick={toggleWidget}>
                        <div className="rhombus">
                            <div className="rhombusContents">
                                <FaCommentMedical className="icon"/>
                            </div>
                        </div>
                        <h3>
                            <FormattedMessage
                                id={"Home.ChatTitle"}
                                defaultMessage={"Chat en ligne"}
                            />
                        </h3>
                        <p>
                            <FormattedMessage
                                id={"Home.ChatDesc"}
                                defaultMessage={"Rentrez en contact avec Emma, notre assistante virtuelle ! Elle vous conseillera sur votre situation et les mesures à prendre."}
                            />
                        </p>
                    </div>

                    <div id={"docImage"}/>

                    <Link to={"/map"} className={"descBox"} id={"map"}>
                        <div className="rhombus">
                            <div className="rhombusContents">
                                <FaMapMarkedAlt className="icon"/>
                            </div>
                        </div>
                        <h3>
                            <FormattedMessage
                                id={"Home.MapTitle"}
                                defaultMessage={"Carte des hôpitaux"}
                            />
                        </h3>
                        <p>
                            <FormattedMessage
                                id={"Home.MapDesc"}
                                defaultMessage={"Notre service vous redirigera automatiquement vers les services de santé les plus proches de chez vous."}
                            />
                        </p>
                    </Link>
                </div>

                <Link to={"/landing"} className={"landingLink"}>
                    <h3>
                        <FormattedMessage
                            id={"Home.LearnMore"}
                            defaultMessage={"En savoir plus"}
                        />
                    </h3>
                </Link>
            </div>

        );
    }

    return (
        <div className={"root"}>
            <Helmet>
                <meta name="description"
                      content="Jamais aller aux urgences n'aura été aussi simple ! Découvrez notre chat en ligne, ainsi que notre carte des hôpitaux qui faciliterons votre démarche."/>
                <meta name="robots" content="index, follow"/>
            </Helmet>
            <div className={"homePage"}>
                <div className={"blueCurveWrapper"}>
                    <svg viewBox="0 0 500 150" className={"blueCurve"} preserveAspectRatio="none">
                        <path
                            d="M-14.39,19.23 C172.40,147.53 320.26,189.95 509.87,12.33 L500.00,0.00 L0.00,0.00 Z"/>
                    </svg>
                </div>
                {displayFeaturesDescriptionBoxes()}

                <ChatBotWidget fullscreen={false}/>
            </div>
        </div>

    );

}
