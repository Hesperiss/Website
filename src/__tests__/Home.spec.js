import React from "react";
import { create } from "react-test-renderer";
import Home from '../Components/Home/Home';

describe("Home page", () => {
  test("Matches the snapshot", () => {
    const home = create(<Home/>);
    expect(home.toJSON()).toMatchSnapshot();
  });
});