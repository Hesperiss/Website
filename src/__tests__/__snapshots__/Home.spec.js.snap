// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page Matches the snapshot 1`] = `
<div>
  <div
    className="descriptionSection"
  >
    <div
      className="textWrapper"
    >
      <h2
        className="sectionTitle"
      >
        Kwili
      </h2>
      <p
        className="sectionSubtitle"
      >
        Jamais aller aux urgences n'aura été aussi simple !
      </p>
    </div>
    <div
      className="descBoxWrapper"
    >
      <button
        className="descBox btn btn-primary"
        disabled={false}
        href=""
        id="chat"
        onClick={[Function]}
        type="button"
      >
        <div
          className="rhombus"
        >
          <div
            className="rhombusContents"
          >
            <svg
              className="icon"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M256 32C114.62 32 0 125.12 0 240c0 49.56 21.41 95 57 130.74C44.46 421.05 2.7 466 2.2 466.5A8 8 0 0 0 8 480c66.26 0 116-31.75 140.6-51.38A304.66 304.66 0 0 0 256 448c141.39 0 256-93.12 256-208S397.39 32 256 32zm96 232a8 8 0 0 1-8 8h-56v56a8 8 0 0 1-8 8h-48a8 8 0 0 1-8-8v-56h-56a8 8 0 0 1-8-8v-48a8 8 0 0 1 8-8h56v-56a8 8 0 0 1 8-8h48a8 8 0 0 1 8 8v56h56a8 8 0 0 1 8 8z"
              />
            </svg>
          </div>
        </div>
        <h3>
          Chat en ligne
        </h3>
        <p>
          Rentrez en contact avec notre CHATBOT, il vous prendra en charge avant même d'être physiquement à l'hôpital.
        </p>
      </button>
      <a
        className="descBox btn btn-primary"
        href="/map"
        id="map"
        onClick={[Function]}
        onKeyDown={[Function]}
      >
        <div
          className="rhombus"
        >
          <div
            className="rhombusContents"
          >
            <svg
              className="icon"
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 576 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M288 0c-69.59 0-126 56.41-126 126 0 56.26 82.35 158.8 113.9 196.02 6.39 7.54 17.82 7.54 24.2 0C331.65 284.8 414 182.26 414 126 414 56.41 357.59 0 288 0zm0 168c-23.2 0-42-18.8-42-42s18.8-42 42-42 42 18.8 42 42-18.8 42-42 42zM20.12 215.95A32.006 32.006 0 0 0 0 245.66v250.32c0 11.32 11.43 19.06 21.94 14.86L160 448V214.92c-8.84-15.98-16.07-31.54-21.25-46.42L20.12 215.95zM288 359.67c-14.07 0-27.38-6.18-36.51-16.96-19.66-23.2-40.57-49.62-59.49-76.72v182l192 64V266c-18.92 27.09-39.82 53.52-59.49 76.72-9.13 10.77-22.44 16.95-36.51 16.95zm266.06-198.51L416 224v288l139.88-55.95A31.996 31.996 0 0 0 576 426.34V176.02c0-11.32-11.43-19.06-21.94-14.86z"
              />
            </svg>
          </div>
        </div>
        <h3>
          Carte des hopitaux
        </h3>
        <p>
          Grace à notre carte des hopitaux, nous vous trouvons automatiquement le meilleur itinéraire pour l'hôpital le plus proche, tout en bénéfiçiant d'informations en temps réel.
        </p>
      </a>
    </div>
  </div>
  <div
    className="introText"
  >
    <a
      href="/landing"
    >
      <button>
        En savoir plus
      </button>
    </a>
  </div>
  <div>
    <div
      className="rcw-widget-container  "
    >
      <button
        className="rcw-launcher"
        onClick={[Function]}
        type="button"
      >
        <img
          alt=""
          className="rcw-open-launcher"
          src="data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz4KPHN2ZyB3aWR0aD0iMzRweCIgaGVpZ2h0PSIzMnB4IiB2aWV3Qm94PSIwIDAgMzQgMzIiIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+CiAgICA8IS0tIEdlbmVyYXRvcjogU2tldGNoIDQ1LjEgKDQzNTA0KSAtIGh0dHA6Ly93d3cuYm9oZW1pYW5jb2RpbmcuY29tL3NrZXRjaCAtLT4KICAgIDx0aXRsZT5pY19idXR0b248L3RpdGxlPgogICAgPGRlc2M+Q3JlYXRlZCB3aXRoIFNrZXRjaC48L2Rlc2M+CiAgICA8ZGVmcz48L2RlZnM+CiAgICA8ZyBpZD0iUGFnZS0xIiBzdHJva2U9Im5vbmUiIHN0cm9rZS13aWR0aD0iMSIgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIj4KICAgICAgICA8ZyBpZD0iRGVza3RvcC1IRCIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoLTEyOTkuMDAwMDAwLCAtNzQ4LjAwMDAwMCkiIGZpbGwtcnVsZT0ibm9uemVybyIgZmlsbD0iI0ZGRkZGRiI+CiAgICAgICAgICAgIDxnIGlkPSJHcm91cC0yIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxMjg5LjAwMDAwMCwgNzM1LjAwMDAwMCkiPgogICAgICAgICAgICAgICAgPGcgaWQ9ImljX2J1dHRvbiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMTAuMjA2OTMzLCAxMy42MDc4MjUpIj4KICAgICAgICAgICAgICAgICAgICA8cGF0aCBkPSJNMjUuNzg2NTM5OCw3LjM3MDUxODYzIEMyNS43ODY1Mzk4LDguNDA4MDMxMTggMjUuMDQzMjAyNiw5LjAzOTAxMDkgMjQuNTAwNDE2MSw5LjUwMjA1NjQyIEMyNC4yOTQ4NTE3LDkuNjc3NDQzODggMjMuOTEyMjI3MSw5Ljk5NzQ2MjMgMjMuOTEzMjQ4NCwxMC4xMTY5OTc1IEMyMy45MTY5NjIzLDEwLjU4NTg5MjQgMjMuNTQ1OTQzNiwxMC45NjAzNDc5IDIzLjA4NDM5ODYsMTAuOTYwMzQ3OSBDMjMuMDgyMDc3NSwxMC45NjAzNDc5IDIzLjA3OTc1NjMsMTAuOTYwMzQ3OSAyMy4wNzc0MzUxLDEwLjk2MDM0NzkgQzIyLjYxOTEzOTgsMTAuOTYwMzQ3OSAyMi4yNDU3MDcxLDEwLjU5NzQwMjUgMjIuMjQxOTkzMiwxMC4xMzA4NjYyIEMyMi4yMzQ2NTgyLDkuMjExMjg0OTcgMjIuODk1MDgyNiw4LjY2MDU5Mjg5IDIzLjQyNTc5ODksOC4yMDc4MzA5OCBDMjMuODQyMjIsNy44NTI1Mjc1MSAyNC4xMTUyODQ2LDcuNjAyNTEzMTEgMjQuMTE1Mjg0Niw3LjM3MzcyNjM2IEMyNC4xMTUyODQ2LDYuOTEwNTg2NDkgMjMuNzQ0NDUxNiw2LjUzMzc3MjM1IDIzLjI4ODY2MzIsNi41MzM3NzIzNSBDMjIuODMyNjg5LDYuNTMzNzcyMzUgMjIuNDYxNzYzMiw2LjkxMDU4NjQ5IDIyLjQ2MTc2MzIsNy4zNzM3MjYzNiBDMjIuNDYxNzYzMiw3Ljg0MjcxNTYyIDIyLjA4NzY4MDYsOC4yMjI4MzE4NCAyMS42MjYxMzU2LDguMjIyODMxODQgQzIxLjE2NDU5MDYsOC4yMjI4MzE4NCAyMC43OTA1MDgsNy44NDI3MTU2MiAyMC43OTA1MDgsNy4zNzM3MjYzNiBDMjAuNzkwNTA4LDUuOTc0MjExODMgMjEuOTExMDg0Niw0LjgzNTU2MTM4IDIzLjI4ODQ3NzUsNC44MzU1NjEzOCBDMjQuNjY1OTYzMiw0LjgzNTQ2NzAzIDI1Ljc4NjUzOTgsNS45NzA5MDk3NiAyNS43ODY1Mzk4LDcuMzcwNTE4NjMgWiBNMjMuMTAwNTU0MSwxMS43NDQxNjY2IEMyMi42MzkwMDkxLDExLjc0NDE2NjYgMjIuMjgzNDAzMSwxMi4xMjQyODI4IDIyLjI4MzQwMzEsMTIuNTkzMjcyMSBMMjIuMjgzNDAzMSwxMi41OTk3ODE5IEMyMi4yODM0MDMxLDEzLjA2ODc3MTIgMjIuNjM5MTAyLDEzLjQ0NTY3OTcgMjMuMTAwNTU0MSwxMy40NDU2Nzk3IEMyMy41NjIwMDYzLDEzLjQ0NTY3OTcgMjMuOTM2MTgxNywxMy4wNjIyNjE0IDIzLjkzNjE4MTcsMTIuNTkzMjcyMSBDMjMuOTM2MTgxNywxMi4xMjQyODI4IDIzLjU2MjA5OTEsMTEuNzQ0MTY2NiAyMy4xMDA1NTQxLDExLjc0NDE2NjYgWiBNNi4yNTcxNzk2LDE3LjY1ODk0MTEgQzUuNzk1NjM0NjIsMTcuNjU4OTQxMSA1LjQyMTU1MTk5LDE4LjAzOTA1NzMgNS40MjE1NTE5OSwxOC41MDgwNDY2IEM1LjQyMTU1MTk5LDE4Ljk3NzAzNTggNS43OTU2MzQ2MiwxOS4zNTcxNTIgNi4yNTcxNzk2LDE5LjM1NzE1MiBMNi4zMjY4MTUyNCwxOS4zNTcxNTIgQzYuNzg4MzYwMjMsMTkuMzU3MTUyIDcuMTYyNDQyODYsMTguOTc3MDM1OCA3LjE2MjQ0Mjg2LDE4LjUwODA0NjYgQzcuMTYyNDQyODYsMTguMDM5MDU3MyA2Ljc4ODM2MDIzLDE3LjY1ODk0MTEgNi4zMjY4MTUyNCwxNy42NTg5NDExIEw2LjI1NzE3OTYsMTcuNjU4OTQxMSBaIE05LjE2MTM1Njk3LDE3LjY1ODk0MTEgQzguNjk5ODExOTgsMTcuNjU4OTQxMSA4LjMyNTcyOTM1LDE4LjAzOTA1NzMgOC4zMjU3MjkzNSwxOC41MDgwNDY2IEM4LjMyNTcyOTM1LDE4Ljk3NzAzNTggOC42OTk4MTE5OCwxOS4zNTcxNTIgOS4xNjEzNTY5NywxOS4zNTcxNTIgTDkuMjMwODk5NzUsMTkuMzU3MTUyIEM5LjY5MjQ0NDc0LDE5LjM1NzE1MiAxMC4wNjY1Mjc0LDE4Ljk3NzAzNTggMTAuMDY2NTI3NCwxOC41MDgwNDY2IEMxMC4wNjY1Mjc0LDE4LjAzOTA1NzMgOS42OTI0NDQ3NCwxNy42NTg5NDExIDkuMjMwODk5NzUsMTcuNjU4OTQxMSBMOS4xNjEzNTY5NywxNy42NTg5NDExIFogTTMzLjcwMzY0NzMsNC4wOTY5MzM5NSBMMzMuNzAzNjQ3MywxMy42NTQxODIyIEMzMy43MDM2NDczLDE1LjkxNzA0ODMgMzEuOTE4MDAzOSwxNy43NTMyODYxIDI5LjY5MTA1NjMsMTcuNzUzMjg2MSBMMTguODU0MTczMSwxNy43NTMyODYxIEMxOC42NDIyOTUxLDE3Ljc1MzI4NjEgMTguNDY3ODM0NiwxNy43NTIwNTk2IDE4LjMyMTg3ODMsMTcuNzQ4OTQ2MyBDMTguMTY4OTU4NSwxNy43NDU3Mzg1IDE3Ljk5NjgxOTIsMTcuNzQ0NTEyIDE3LjkyOTY5MDQsMTcuNzUyMDU5NiBDMTcuODU5Nzc2MiwxNy43OTkyMzIyIDE3LjY4ODM3OTcsMTcuOTU4NzY5NyAxNy41MjMyMDQsMTguMTEzNDAxMiBDMTcuNDU3NzQ2NSwxOC4xNzQ2MzExIDE3LjM4MzY1NDIsMTguMjQxOTkzNSAxNy4zMDM3MTI1LDE4LjMxNjE0ODcgTDE0LjIzNzcwMTksMjEuMTU1NzQ2MSBDMTMuOTkzMTQxNSwyMS4zODI0NTczIDEzLjYzMTMxNDgsMjEuNDQxMTM5OSAxMy4zMjg0NDYyLDIxLjMwNTQ3MTcgQzEzLjAyNTU3NzYsMjEuMTY5NzA5MiAxMi44MTI5NTY4LDIwLjg2NDk3NDcgMTIuODEyOTU2OCwyMC41Mjg2MzQ2IEwxMi44MTI5NTY4LDExLjUyNjUxMjYgTDQuMDU5MjkzMjcsMTEuNTI2NTEyNiBDMi43NTM2NzE1NCwxMS41MjY1MTI2IDEuNjcxMjU1MjQsMTIuNjEzNjUwNiAxLjY3MTI1NTI0LDEzLjk0MDE0MjEgTDEuNjcxMjU1MjQsMjMuNDk3MzkwNCBDMS42NzEyNTUyNCwyNC44MjM5NzYyIDIuNzUzNzY0MzksMjUuODY2OTYwNyA0LjA1OTI5MzI3LDI1Ljg2Njk2MDcgTDE2LjE2NDE5NDksMjUuODY2OTYwNyBDMTYuMzcyNjM3NiwyNS44NjY5NjA3IDE2LjU2NzYxNzQsMjUuOTY0MjMwNSAxNi43MjE1NTg2LDI2LjEwNzA2ODkgTDE5LjIxOTQzNTIsMjguNDQ3MjAzNiBMMTkuMjE5NDM1MiwyMC4zNzcwMjIxIEMxOS4yMTk0MzUyLDE5LjkwODAzMjggMTkuNTkzNTE3OCwxOS41Mjc5MTY2IDIwLjA1NTA2MjgsMTkuNTI3OTE2NiBDMjAuNTE2NjA3OCwxOS41Mjc5MTY2IDIwLjg5MDY5MDUsMTkuOTA4MDMyOCAyMC44OTA2OTA1LDIwLjM3NzAyMjEgTDIwLjg5MDY5MDUsMzAuMzcxNzQ4NCBDMjAuODkwNjkwNSwzMC43MDgwODg1IDIwLjcwMTI4MTUsMzEuMDA4Mjk0NCAyMC4zOTgzMjAxLDMxLjE0NDA1NyBDMjAuMjg5OTY3LDMxLjE5MjY0NDcgMjAuMTgxMTQ5OCwzMS4yMTE3OTY3IDIwLjA2NzIyNTksMzEuMjExNzk2NyBDMTkuODYyNTg5OSwzMS4yMTE3OTY3IDE5LjY2MDczOTQsMzEuMTI2NDE0NCAxOS41MDM3MzQzLDMwLjk4MDc0NTcgTDE1Ljg0MDI1LDI3LjU2NTA3NzQgTDQuMDU5MjkzMjcsMjcuNTY1MDc3NCBDMS44MzIyNTI4MiwyNy41NjUwNzc0IDAsMjUuNzYwMjU2NSAwLDIzLjQ5NzI5NiBMMCwxMy45NDAxNDIxIEMwLDExLjY3NzI3NiAxLjgzMjI1MjgyLDkuODI4MzAxNjIgNC4wNTkyOTMyNyw5LjgyODMwMTYyIEwxMi44MTI5NTY4LDkuODI4MzAxNjIgTDEyLjgxMjk1NjgsNC4wOTY5MzM5NSBDMTIuODEyOTU2OCwxLjgzMzk3MzUgMTQuNjU5MjI5NiwwLjAxNjQxNjAzOTMgMTYuODg2MjcwMSwwLjAxNjQxNjAzOTMgTDI5LjY5MTE0OTEsMC4wMTY0MTYwMzkzIEMzMS45MTgwMDM5LDAuMDE2NDE2MDM5MyAzMy43MDM2NDczLDEuODMzOTczNSAzMy43MDM2NDczLDQuMDk2OTMzOTUgWiBNMzIuMDMyMzkyLDQuMDk2OTMzOTUgQzMyLjAzMjM5MiwyLjc3MDM0ODE2IDMwLjk5NjQ5MjMsMS43MTQ2MjcgMjkuNjkxMDU2MywxLjcxNDYyNyBMMTYuODg2MjcwMSwxLjcxNDYyNyBDMTUuNTgwNzQxMiwxLjcxNDYyNyAxNC40ODQyMTIsMi43NzAzNDgxNiAxNC40ODQyMTIsNC4wOTY5MzM5NSBMMTQuNDg0MjEyLDEwLjY4NTMzMjEgTDE0LjQ4NDIxMiwxOC42MDQwODk4IEwxNi4xNjQyODc4LDE3LjA2MTI2NTIgQzE2LjI0MjA5NCwxNi45ODkwOTEyIDE2LjMyOTI3NzgsMTYuOTIzMDQ5NyAxNi4zOTI4Nzg0LDE2Ljg2MzQyMzYgQzE3LjE2MDA3NzQsMTYuMTQ1MDgwMyAxNy4zNTYzNTcsMTYuMDMwMjYyNCAxOC4zNTcwNjc1LDE2LjA1MTIwNyBDMTguNDkzNTUzNCwxNi4wNTQxMzE3IDE4LjY1NjEyOTQsMTYuMDU1MTY5NSAxOC44NTQxNzMxLDE2LjA1NTE2OTUgTDI5LjY5MTA1NjMsMTYuMDU1MTY5NSBDMzAuOTk2NDkyMywxNi4wNTUxNjk1IDMyLjAzMjM5MiwxNC45ODA3NjggMzIuMDMyMzkyLDEzLjY1NDI3NjYgTDMyLjAzMjM5Miw0LjA5NjkzMzk1IFoiIGlkPSJTaGFwZSI+PC9wYXRoPgogICAgICAgICAgICAgICAgPC9nPgogICAgICAgICAgICA8L2c+CiAgICAgICAgPC9nPgogICAgPC9nPgo8L3N2Zz4="
        />
      </button>
    </div>
  </div>
</div>
`;
